transit_odp/pipelines/tests/test_dataset_etl/test_etl_operating_profiles.py

```diff
@@ -0,0 +1,202 @@
+from dateutil import tz
```

Comment:
Can we add a test case to read the same from vehicle journey element?

Reply:
Done